### PR TITLE
docs(ru): Translate directory features

### PR DESCRIPTION
**Pull request by zavvla**
Added translate "features/deployment-targets" for RU

### CODE REVIEW

**Comment by zavvla** [comment]
@DreaMinder make a review please

**Review by DreaMinder — CHANGES_REQUESTED** [review]
Hey @zavvla, nice work.
I noticed you added line breaks on every sentence comparing to the eng version. Is that intentional? Can we discuss that part?

**Comment by zavvla** [comment]
Hi @DreaMinder.
Yes, this is intentional. In my opinion, it is easier to read text that does not have a codebase(pure code) this way. If something is wrong, please fix it.

File: content/ru/guides/features/deployment-targets.md [hunk 1]
```diff
@@ -0,0 +1,62 @@
...
+title: Публикация приложения на хостинге
+description: Публикация приложения на хостинге
+position: 2
+category: features
+---
+
+## Статический хостинг
+
+Nuxt.js также работает, как генератор статических сайтов.
+Сгенерировав статическую версию Nuxt.js приложения, вы получаете все преимущества универсального приложения(SPA) работающего без сервера.
```
Comment (DreaMinder) [inline content/ru/guides/features/deployment-targets.md:11]:
```suggestion
Сгенерировав статическую версию Nuxt.js приложения, вы получаете все преимущества универсального приложения (SPA) работающего без сервера.
```

**Review by DreaMinder — APPROVED** [review]

**Pull request merged**